elasticsearch-cluster-CCR/README.md

```diff
@@ -0,0 +1,77 @@
+# Configuring CCR (Cross Cluster Replication) on Elasticsearch
+
+To make this configuration we must have 2 elasticsearchs clusters:</br>
+
+ - **The Leader**
+ - **The Follower**
+
+An example of deployment is in the `./elasticsearch`, to deploy it just run the next command within your specific cluster and namespace:</br>
+
+```bash
+kubectl apply -f ./elasticsearch
+```
+
+Both the leader and the follower can share the same configuration, which is why there is only one elasticsearch cluster deployment configuration within `./elasticsearch`.</br>
+
+After deploying the elasticsearch cluster in each environment and also the external IPs, we will use it to configure the CCR between them using the Elasticsearch API.</br>
+
+First, for CCR configuration, we need to configure the **Remote Cluster**, that should be done in Follower Cluster to Leader Cluster.</br>
+
+Assuming my elasticsearch Leader Cluster is accessible by `http://leader-elasticsearch-ip:9200` and the follower is accessible by `http://folower-elasticsearch-ip:9200`</br>
```

Comment:
Use on the examples the service names used on the YAML files, or if it needs to be an IP address, replace `leader-elasticsearch-ip` and `follower-elasticsearch-ip` by environment variables, and explain the expected values